src/components/invest/investslug/components/Header.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+interface HeaderProps {
+  title: string
+  subtitle: string
+}
+
+const Header: React.FC<HeaderProps> = ({ title, subtitle }) => {
+  return (
+    <div className='flex flex-col gap-10 h-[150px]'>
```

Comment:
no need to define height here I believe.
We can just use flex with proper gap (gap-8 I think)